css/styles.css

```diff
@@ -0,0 +1,126 @@
+* { 
+  box-sizing: border-box; 
+}  
+
+h1{
+  color: #fdf65e;
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  margin:0px;
+  font-size:40px;
+}
+
+nav{
+  
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  /*float: right;*/
+  display:block;
+  text-align: right;
+  border-bottom: #fdf65e solid 4px ;
+}
+a{
+  color:#fff;
+  font-size:30px;
+  text-decoration: none;
+  margin-right: 5px;
+}
+.speciallink{
+  font-style: bold;
+  font-family: impact;
+}
+
+h2{
+  text-align: center;
+  font-style: italic;
+  font-size:30px;
+  background-color: #c6c6c6;
+  margin:0px;
+  padding: 12px 12px 12px 12px;
```

Comment:
@akallbold since all for side are equal this code have been written with a short cut `padding: 12px`

Reply:
Removed extra px clarifications